Brickwall/__init__.py

```diff
@@ -1,53 +1,180 @@
-from flask import Flask
+from flask import Flask, request
 from flask_restplus import Api, Resource, fields
 from flask_sqlalchemy import SQLAlchemy
+import os
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///test.db'
-api = Api(app = app,
-    version="0.0",
-    title= "BrickWall",
-    description="The Backend of the Brickwall webapp")
-name_space = api.namespace('main', description='Main APIs')
+
+# Load default configuration and any environment variable overrides
+_root_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+app.config.from_pyfile(os.path.join(_root_dir, 'config.env.py'))
+
+# Load file based configuration overrides if present
+_pyfile_config = os.path.join(_root_dir, 'config.py')
+if os.path.exists(_pyfile_config):
+    app.config.from_pyfile(_pyfile_config)
+
+api = Api(app=app,
+          version="0.0",
+          title="BrickWall",
+          description="The Backend of the BrickWall Web App")
+v1 = api.namespace('api/v1', description='v1 APIs')
 db = SQLAlchemy(app)
 
-@name_space.route('/api/v1/location/<location_id>')
-class Location(Resource):
+from Brickwall.models import *
+db.create_all()
+
+location_model = api.model('Location', {
+    'name': fields.String
+})
+
+review_model = api.model('Review', {
```

Comment:
This already exists inside the `Review` model.
The `get_model(cls, api)` method returns this already. This is unnecessary